easybuild/easyconfigs/t/TensorFlow/tensorflow-v1.3.0-cuda-python3-cscs.patch

```diff
@@ -27,8 +27,8 @@ index 0000000..11f0357
 +
 +CONF_IN="configure.in"
 +
-+echo $EBROOTPYTHON"/bin/python"$PYMAJVER > $CONF_IN
-+echo $EBROOTPYTHON"/lib/python"$PYMAJVER"."$PYMINVER"/site-packages" >> $CONF_IN
++echo "/opt/python/$PYMAJVER.$PYMINVER.$PYREVVER/bin/python"$PYMAJVER > $CONF_IN
```

Comment:
I would suggest to use the EB variable: `$EBROOTCRAYPYTHON` 

Here and in the other patches as well.

Reply:
`cray-python/3.6.1.1` module doesn't define `$PYTHON_PATH` so the `$EBROOTPYTHON` variable is not created automatically by EasyBuild. It stopped working when moving from `cray-python/17.06.1` which do define `$PYTHON_PATH`.

The SWIG-3.0.10-CrayGNU-17.08-python3 and -python3.6 easyconfigs also use
`$EBROOTCRAYPYTHON` for a `configopts`, which seems to be ignored. @lucamar and @gppezzi should we replace there also `$EBROOTCRAYPYTHON` by `/opt/python/$PYMAJVER.$PYMINVER.$PYREVVER/`? Or is there a better solution?

Reply:
Just for the record: we should continue to use `$EBROOTPYTHON`, since `Python` is the package name taken from the metadata file (`$EBROOTCRAYPYTHON` should not be used).

We're going to ask Cray to add a variable pointing to "python root" on their `cray-python` module file, in order to avoid hardcoding the path in the metadata file in the future.